.github/workflows/test.yml

```diff
@@ -27,7 +27,7 @@ jobs:
         uses: golangci/golangci-lint-action@v3
         with:
           args: --timeout 3m
-          version: latest
+          version: v1.51.2
```

Comment:
Note: This change is to avoid changing the source just for that PR.
golangci-lint@1.52.0 introduces new linting errors.

Another PR should be made for this.